gocd/xcdchk.jump.opensuse.yaml

```diff
@@ -0,0 +1,21 @@
+format_version: 3
+pipelines:
+  openSUSE.QA.Maint:
```

Comment:
this pipeline is something else

Reply:
I did use a separate pipeline

  XCDCHK.openSUSE_Jump:
    group: openSUSE.XCDCHK
    lock_behavior: unlockWhenFinished

Does the group look like a good match?